### PR TITLE
Add answers to the Security and Privacy self-review

### PR DESCRIPTION
**Pull request by beverloo**
See https://www.w3.org/TR/security-privacy-questionnaire/. It seems easy enough to just check this in.

/cc @LJWatson

### CODE REVIEW

**Comment by LJWatson** [comment]
Thanks @Beverloo. LGTM.

When this is merged, I'll ping the Security and Privacy IGs for a review (tracked via #265).

File: security-privacy-questionnaire.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+## 3.8 Does this specification allow an origin access to sensors on a user’s device?
+
+No.
+
+## 3.9 Does this specification allow an origin access to aspects of a user’s local computing environment?
+
+Yes, the push service selected by the device. In some cases, most commonly on
+desktop platforms, a user agent includes a push service client separate from the
+push service made available by the operating system.
```
Comment (martinthomson) [inline security-privacy-questionnaire.md]: I'm not sure what "access" means here.  The specification allows an origin to identify the push service.  I would say that the push service is selected by the user agent and that that might use the same push service as other applications.

**Review by martinthomson — APPROVED** [review]
Not sure about the interpretation of the one question, but I'm sure that this will do.

File: security-privacy-questionnaire.md (hunk 1 above)
Comment (beverloo) [reply security-privacy-questionnaire.md]: Updated, thanks!

**Pull request merged**